docs/Dependency-injection.md

```diff
@@ -49,7 +49,56 @@ public class EmployeesController {
 	// use _mapper.Map or _mapper.ProjectTo
 }
 ```
+### AutoFac
 
+https://dotnetfalcon.com/autofac-support-for-automapper/
```

Comment:
I don't know anything about AutoFac, but LGTM :) It seems fair to have the link. I didn't ask for permission yet, but I will.

Reply:
Thanks for reaching out to me. Go ahead and add it to the official documentation.

Reply:
Thanks!